fmf/utils.py

```diff
@@ -419,7 +422,7 @@ class Logging:
     _level = LOG_WARN
 
     # Already initialized loggers by their name
-    _loggers = dict()
+    _loggers: dict[str, logging.Logger] = dict()
```

Comment:
This doesn't have `__future__.annotations` here so it should be using `Dict` instead here. But also let's deffer the change.

Reply:
I think `dict` is already in Python 3.9, no?

Reply:
Correct, Python 3.9 allows builtin types as annotations.

Reply:
Ah right, must have confused it with `Union` compatibility issue